src/tools/scripts/update_timestamps_versions_on_commit.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+######################################
+## Custom Web Almanac script        ##
+######################################
+#
+# This script updates CSS and JS reference to new versions numbers
+# when they are included in a commit. This is used for cahce busting.
+# 
+# It also updates timestamps for any .md and .html files changed
+#
+# This script is run in a GitHub Action and should not need to be run manually
+#
+# Note:  we use basic sed with no support for "+" (one or more) and instead
+# fake it with syntax like "[0-9][0-9]*" to avoid having to enable extended
```

Comment:
Will have a think. Using regexs different than the standard / is also confusing to future contributors.

Reply:
The `/` and alternative characters in sed are separators, not part of the regex and their purpose is precisely to avoid escaping as `/` is a common thing in Unix file paths. That said, we can have inline comments right above such sed commands to say something like, `# using | instead of / for separator to avoid unnecessary escaping`. If we minimize the number of sed commands (for which I have some notes) then this will not be a bad idea., Also, we should be able to use more compact and readable RegExes that way.

Reply:
Actually now this has been refactored so it doesn't use the paths it's no more difficult to use extended regex. So have enabled it to allow us to use `+`. Resolving.